src/EventList.js

```diff
@@ -108,6 +119,44 @@ export function EventList(props) {
     </div>
     </ul>
   );
+   *
+   */
+
+  /* ??? frm: HACK - requires work:
+   *
+   * The code below is a modified version that attempts to put the kicker in as one of the list elements
```

Comment:
What are the downsides to the change as it currently exists?

Reply:
There is no functional problem - the UI/UX as it is with my changes is just a little more pedestrian than what was there before.  

I was mucking around with CSS and digging a hole for myself.  This is just one side-effect of that mucking around.  I will see if I can revert these changes and deal with the CSS issues in a separate PR.